lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -48,21 +55,111 @@ def _create_store(self):
         with self.conn:
             self.conn.execute("CREATE TABLE IF NOT EXISTS subscriberdb"
                               "(subscriber_id text PRIMARY KEY, data text)")
+        with self.conn:
+            self.conn.execute(
+                "CREATE TABLE IF NOT EXISTS apndb"
+                "(apn_name text PRIMARY KEY, data text, subscriberids text)"
+            )
```

Comment:
From Tariq: "
Looks like "data" is a serialized string of "SubscriberData"
Not sure why is that the case!
Shouldn't this be a serialized string of "APNConfiguration"
"
I somehow missed this, indeed you should have used APNConfiguration...